README.md

```diff
@@ -334,8 +334,18 @@ TODO:   - Debian https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=TBDSRC
 TODO:   - Upstream's bug tracker, e.g., GitHub Issues
 TODO: - The package has important open bugs, listing them: TBD
 TODO-A: - The package does not deal with exotic hardware we cannot support
-TODO-B: - The package does deal with exotic hardware, it is present at TBD
-TODO-B:   to be able to test, fix and verify bugs
+TODO-B: - The package does deal with exotic hardware, such hardware is available
+TODO-B:   to the team for debugging, test, verification and development via
```

Comment:
Nitpick, a `:` at the end would be nice.

Reply:
discussed and resolved in a follow on commit